README.md

```diff
@@ -72,7 +72,7 @@ To build the project use the command:
   * Assemble the main classes  
     * `./gradlew classes`    
   * Assemble and test the project
-    * `./gradlew build`   
+    * `./gradlew build -x test`   
```

Comment:
Why is it necessary to exclude `test` from the build command in order to "Assemble and test the project"?

Reply:
We were getting an error where the filters list was not showing up unless we excluded the test from that command, but after running through it again it seems to work so the "-x test" is actually not needed and we will remove it.